style.css

```
@@ -1 +1,260 @@
-/* 본인의 디자인 감각을 최대한 발휘해주세요! */
+@import url('../reset.css');
+
+@font-face {
+  font-family: 'omyu_pretty';
+  font-style: normal;
+  font-weight: normal;
+  src: url('https://cdn.jsdelivr.net/gh/projectnoonnu/noonfonts_2304-01@1.0/omyu_pretty.woff2') format('woff2');
+}
+
+html {
+  min-width: 375px;
+}
+
+body {
+  font-family: 'omyu_pretty';
+}
+
+header {
+  padding: 1.5rem;
+  border-bottom: 1px solid black;
+  background-color: rgb(134 196 173);
+  text-align: center;
+}
+
+h1 {
+  font-size: 2rem;
+  font-weight: bold;
+}
+
+/* history section */
+.history {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+  margin: 1rem 3rem;
+  padding: 1rem;
+  border: 3px solid black;
+  border-radius: 0.7rem;
+  background-color: rgb(134 196 173);
+}
+
+.history > h2 {
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+.detail {
+  display: flex;
+  gap: 0.7rem;
+  align-items: center;
+  margin: 0.7rem;
+}
+
+.detail i {
+  display: inline-block;
+  border-radius: 2rem;
+  font-size: 1.2rem;
+  font-weight: 600;
+  text-align: center;
+}
+
+.history span:nth-child(2),
+i:nth-child(1) {
+  color: blue;
+}
+
+.history span:nth-child(4),
+i:nth-child(3) {
+  color: red;
+}
+
+.schedule i {
+  color: blue;
+}
+
+.done i {
+  color: red;
+}
+
+/* edit section */
+.edit {
+  display: flex;
+  flex-direction: column;
+}
+
+/* date nav */
+.date {
+  display: flex;
+  gap: 2rem;
+  justify-content: center;
+  margin-top: 1rem;
+  font-size: 1.5rem;
+}
+
+.list {
+  width: 100%;
+}
+
+.trash {
+  padding: 0.3rem 1rem;
+  border: 1px solid rgb(33 109 176);
+  border-radius: 0.5rem;
+  background-color: white;
+  font-family: 'omyu_pretty';
+  color: rgb(33 109 176);
```

여러 곳에서 동일한 색상 코드와 폰트 스타일이 반복적으로 사용되고 있는 거 같습니다.
아래와 같이 변수로 관리하는 것도 좋은 방법인 것 같아요
```suggestion
:root {
  --primary-color: rgb(134, 196, 173);
  --secondary-color: rgb(33, 109, 176);
  --font-family: 'omyu_pretty';
}
```